packages/wallets/src/features/cashier/modules/DepositFiat/DepositFiat.tsx

```diff
@@ -19,13 +26,15 @@ const DepositFiat = () => {
 
     return (
         <React.Fragment>
-            {isLoading && <Loader />}
-            {iframeUrl && !isLoading && (
+            {(isAddressLoading || isContentLoading) && <Loader />}
+            {iframeUrl && !isAddressLoading && (
```

Comment:
@lubega-deriv 
believe that we can remove `isAddressLoading` from useCashierFiatAddress hook. 
since we are defining `isContentLoading` as true during the first mount and only when iframe loaded we set it to false we can remove isAddressLoading check.

Reply:
Updated implementation 👍